### PR TITLE
Clean up GCP disks during deprovision

### PR DESCRIPTION
**Pull request by ironcladlou**
Find and delete disks associated with GCP test clusters during deprovisioning.

### CODE REVIEW

File: roles/openshift_gcp/templates/remove.j2.sh [hunk 1]
```diff
@@ -175,4 +175,17 @@ done
 # Network
 ( teardown "{{ openshift_gcp_network_name }}" compute networks ) &
 
+# Disks
+(
+    if ! disks=$( gcloud --project "{{ openshift_gcp_project }}" compute disks list --filter="users~projects/{{ openshift_gcp_project }}/zones/{{ openshift_gcp_zone }}/instances/{{ openshift_gcp_prefix }}-.*" --format="value[terminator=' '](name)" 2>/dev/null ); then
```
Comment (ironcladlou) [inline roles/openshift_gcp/templates/remove.j2.sh]: Hoping this regex is good enough...

Comment (smarterclayton) [reply roles/openshift_gcp/templates/remove.j2.sh]: we'll find out

**Pull request merged**